### PR TITLE
feat: without second confirmation when the data of issue drawer has no change

### PR DESCRIPTION
**Pull request by sherotree**
## What type of PR is this?

- [x] Feature
- [ ] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [ ] Chore

## What this PR does / why we need it:
 create the new issue,  when the data of issue drawer has no change, remove second confirmation and close directly

## Which issue(s) this PR fixes:

Fixes #

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [ ] No

### CODE REVIEW

File: shell/app/modules/project/types/issue.d.ts [hunk 1]
```diff
@@ -237,3 +237,14 @@ declare namespace ISSUE {
   }
 }
 
+interface DrawerData {
+  [k: string]: any;
+  assignee: string;
+  bugStage: 'demandDesign';
```
Comment (McDaddy) [inline shell/app/modules/project/types/issue.d.ts:243]: fixed value field? if value is fixed then you can push backend developer to ignore it

File: shell/app/modules/project/types/issue.d.ts [hunk 2]
```diff
@@ -237,3 +237,14 @@ declare namespace ISSUE {
   }
 }
 
+interface DrawerData {
+  [k: string]: any;
```
Comment (McDaddy) [inline shell/app/modules/project/types/issue.d.ts:241]: this type can't be enumerate？

File: shell/app/modules/project/types/issue.d.ts [hunk 3]
```diff
@@ -237,3 +237,14 @@ declare namespace ISSUE {
...
+  assignee: string;
+  bugStage: 'demandDesign';
+  priority: 'URGENT' | 'HIGH' | 'NORMAL' | 'LOW';
+  complexity: 'DIFFICULT' | 'NORMAL' | 'EASY';
+  iterationID: number;
+  planFinishedAt: string | undefined;
```
Comment (McDaddy) [inline shell/app/modules/project/types/issue.d.ts:247]: `planFinishedAt?: string`

File: shell/app/modules/project/types/issue.d.ts (hunk 2 above)
Comment (sherotree) [reply shell/app/modules/project/types/issue.d.ts:241]: May have these fields: labels, source, title, content, and custom fields.  The number of fields is not fixed

Comment (McDaddy) [reply shell/app/modules/project/types/issue.d.ts:241]: u should list all of them once u can count it. If there are custom fields, should maintain in a separate field names `customFields`

File: shell/app/modules/project/common/components/issue/issue-drawer.tsx [hunk 4]
```diff
@@ -51,15 +54,43 @@ interface IProps {
...
   const [
     title = IssueDrawer.Empty,
     main = IssueDrawer.Empty,
     tabs = IssueDrawer.Empty,
     meta = IssueDrawer.Empty,
     footer = IssueDrawer.Empty,
   ] = React.Children.toArray(children);
+  const customFieldDetail = issueStore.useStore((s) => s.customFieldDetail);
   const [copyTitle, setCopyTitle] = React.useState('');
+  const [isChanged, setIsChanged] = React.useState(true);
```
Comment (daskyrk) [inline shell/app/modules/project/common/components/issue/issue-drawer.tsx:67]: isChanged default should be false?

**Review by daskyrk — APPROVED** [review]

**Pull request merged**